src/drive/web/modules/views/Create/CreateFromFlagship.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
```

Comment:
should be more in `upload` than `Create`.

Reply:
updated, will replace 
```javascript
      {__TARGET__ === 'mobile' && (
        <Route path="uploadfrommobile" element={<UploadFromMobile />} />
      )}
 ```
 in subsequent pull request